README.md

```diff
@@ -52,7 +52,7 @@ module "eg_prod_bastion_label" {
 }
 ```
 
-This will create an `id` with the value of `eg-prod-bastion-public`.
+This will create an `id` with the value of `eg_prod_bastion_label`.
```

Comment:
Hrm... so I think what we're trying to say is that the value of `eg_prod_bastion_label.id` is `eg-prod-bastion-public` because `namespace=eg`, `stage=prod`, `name=bastion` and `attributes=public`. 

Perhaps there's a better way to say this or explain it?

Reply:
i think that's an excellent explanation.

Perhaps,
"this will generate an `id` with the value of `eg-prod-bastion-public` based off the key:value pairs above."

or any version thereof. my point is that line would benefit from further elaboration.

thanks for clarifying.